packages/bpk-docs/src/pages/ChipsPage/ChipsPage.js

```diff
@@ -44,22 +46,24 @@ class DefaultChipContainer extends Component<{}, State> {
   }
 
   addRemovableChip = () => {
-    this.setState({
+    this.setState(() => ({
       chipNames: [
         ...this.state.chipNames,
         `Example Chip ${this.state.nextChipId}`,
       ],
       nextChipId: this.state.nextChipId + 1,
-    });
+    }));
```

Comment:
If we're using functional `setState` here, would it not be better to do this:

```
this.setState((state) => ({
  chipNames: [
    ...state.chipNames,
   `Example Chip ${state.nextChipId}`,
  ],
  nextChipId: state.nextChipId + 1,
}));
```

Not a huge difference, but it just means we can use the `state` object that's passed in for free and not have to use `this`.